quickemu

```diff
@@ -377,13 +383,30 @@ function vm_boot() {
    fi
 }
 
+function shortcut_create {
+  local VMNAME=$(basename "${VM}" .conf)
+  local LAUNCHER_DIR="$(dirname "$(realpath "$0")")"
+  local filename="/home/${USER}/.local/share/applications/${VMNAME}.desktop"
+  cat << EOF > ${filename}
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=true
+Exec=${LAUNCHER_DIR}/${LAUNCHER} --vm ${VM}
+Name=${VMNAME}
+Icon=/snap/qemu-virgil/74/meta/gui/icon.png
```

Comment:
Icon=/snap/qemu-virgil/74/meta/gui/icon.png
Please change to:
Icon=/snap/qemu-virgil/**current**/meta/gui/icon.png